src/ec/packages/ec-component-site-header-core/site-header-core.story.js

```diff
@@ -19,6 +19,7 @@ storiesOf('Components/Site Headers/Core', module)
     () => {
```

Comment:
maybe we need also a "logged in" story like ECL and standardised component? 
Also by checking the site-headers-standardised stories. 
We have problem with de default story, it seems we are always logged in.

Reply:
fixed